manual/googlechrome-extensions/googlechrome-github-expandinizr/tools/install_x64.reg

```diff
@@ -0,0 +1,2 @@
+[HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432node\Google\Chrome\Extensions\cbehdjjcilgnejbpnjhobkiiggkedfib]
```

Comment:
Interesting. All my packages work as "Wow6432node." I wonder if there is a Windows version difference? I wouldn't think so though.